ocs_ci/deployment/baremetal.py

```diff
@@ -1112,6 +1112,15 @@ def deploy(self, log_cli_level="DEBUG"):
                         ignore_error=True,
                     )
 
+            self.test_cluster()
+            if config.ENV_DATA.get("skip_disks_cleanup", False):
+                logger.info("Skipping disks cleanup")
+            else:
+                logger.info("Performing Disk cleanup")
+                workers = get_nodes(node_type="worker")
+                for worker in workers:
+                    clean_disk(worker)
+
```

Comment:
can we include restarting workaround-ssd

Reply:
@vavuthu all the disk attached to IBM_BM are SSD

Reply:
@prsurve we saw ( IBM BM02 ), after cleaning the disks, rotational value is showing as 1 ( HDD )